src/app/products/recurring-deposit-products/recurring-deposit-product-stepper/recurring-deposit-product-currency-step/recurring-deposit-product-currency-step.component.html

```
@@ -42,19 +42,31 @@
-        matTooltip="{{ 'tooltips.Enter multiples of currency value' | translate }}"
-      />
-      <mat-error>
-        {{ 'labels.inputs.Currency in multiples of' | translate }} {{ 'labels.commons.is' | translate }}
-        <strong>{{ 'labels.commons.required' | translate }}</strong>
-      </mat-error>
-    </mat-form-field>
+    <mat-checkbox
+      labelPosition="before"
+      formControlName="setMultiples"
+      class="flex-98 margin-t"
+      style="align-items: center"
```

HTMLHint (1.8.0)

[warning] Inline style [ 
      style="align-items: center" ] cannot be used.

(inline-style-disabled)